node/src/accountant/scanners/scan_schedulers.rs

```diff
@@ -538,15 +553,76 @@ mod tests {
         let now = UNIX_EPOCH
             .checked_add(Duration::from_secs(1_000_000))
             .unwrap();
-        let subject = NewPayableScanDynIntervalComputerReal::default();
+        let mut subject = make_subject();
+        subject.clock = Box::new(SimpleClockMock::default().now_result(now));
+        subject.last_scan_timestamp = now.checked_add(Duration::from_secs(1)).unwrap();
 
-        let _ = subject.compute_interval(
-            now.checked_sub(Duration::from_secs(1)).unwrap(),
-            now,
-            Duration::from_secs(32),
+        let _ = subject.compute_interval();
+    }
+
+    #[test]
+    fn zero_out_works_for_default_subject() {
```

Comment:
After you rename this function from `zero_out`, please rename this test and the test below.